pkg/gktest/printer_go_test.go

```diff
@@ -0,0 +1,257 @@
+package gktest
+
+import (
+	"errors"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestPrinterGo_Print(t *testing.T) {
+	// Not final. While this will eventually exactly match the output of go test,
```

Comment:
nit: add a TODO

Also, what's different about the output? Does the extra output rely on as-yet missing functionality?

Reply:
I don't know yet. It's too early to do the full-day deep-dive it would probably take for me to fully grok. I'm not going to bother with that sort of polish until we're closer to alpha release. Until we're actually running the tests, timing them & etc it isn't worth this.

The full set of differences would take several paragraphs to write and require hours to research. I don't think it is a good use of time to specify this full difference every time we get closer. It should suffice to say "the Go Printer's output will eventually exactly match the output in the `go test` specification, and the research on the full specification is to be done later".

Reply:
Also, added TODO.